README.md

```diff
@@ -1,11 +1,12 @@
 ## MongoDB Atlas CLI
 
 <p align="center">
-  <img src="https://user-images.githubusercontent.com/5663078/161264663-6c059196-6c20-4efc-980a-6266f7017917.png" alt="CLI Logo">
+    <picture align="center">
+        <source srcset="https://user-images.githubusercontent.com/5663078/161268959-797863f3-f8c7-479b-b40d-4eab662de658.png" media="(prefers-color-scheme: dark)">
+        <img src="https://user-images.githubusercontent.com/5663078/161264663-6c059196-6c20-4efc-980a-6266f7017917.png">
+    </picture>
```

Comment:
doesn't seem to work but found some GH specific alternative https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to